_posts/2022-12-06_Jabref-Birthday-Nikolaus.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  JabRef's 19th Birthday
+id: JabRefBirthday22
+author: "[JabRef Maintainers](https://github.com/JabRef/jabref/blob/main/MAINTAINERS)"
+bg: jabref-font
+color: white 
+---
+
+Last week, on November 29, it was JabRef's 19 birthday! Yep. That old already 😉. While the original creators Morten O. Alver and Nizar Batada are no longer active developing at JabRef, they are the ones who led the foundation in 2003 for the basis of JabRef. And even though they are no longer on board, we have a couple of users who still use JabRef since the early days!
```

Comment:
```suggestion
Last week, on November 29, it was JabRef's 19 birthday! Yep. That old already 😉. While the original creators Morten O. Alver and Nizar Batada are no longer actively developing JabRef, they led the foundations in 2003. Moreover, we have a couple of users who use JabRef since the early days!
```